sql/driver/example_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 func Example() {
```

Comment:
Is this function used 🤔 anywhere?  Also why add random token and cluster name?

Reply:
That example appears in the godoc. They are just placeholders.

Reply:
I did not see it, where should it appear? I use godoc and went to http://localhost:6060/pkg/github.com/hazelcast/hazelcast-go-client/sql/driver/